packages/devextreme/js/core/errors.js

```diff
@@ -249,5 +249,27 @@ export default errorUtils({
     /**
     * @name ErrorsCore.W0018
     */
-    W0018: 'Setting the "position" property with a function is deprecated since v21.2'
+    W0018: 'Setting the "position" property with a function is deprecated since v21.2',
+    /**
+    * @name ErrorsCore.W0019
+    */
+    W0019: 'DevExtreme: Valid license key not found.\n\n' +
+        'If you are using a trial version, you must uninstall all copies of DevExtreme once your 30 days trial period expires. For licensing-related information, please refer to the DevExtreme End User License Agreement: https://js.devexpress.com/EULAs/DevExtremeComplete/.\n\n' +
+        'To continue using DevExtreme in a project, you must purchase a license. For pricing/licensing options, please visit: https://js.devexpress.com/Buy/.\n\n' +
+        'If you have licensing-related questions or need help with a purchase, please email clientservices@devexpress.com. We will be happy to follow-up.',
```

Comment:
Seems like we need to add an additional `\n\n` at the end to separate a default `See: ->link_to_error<-` message with an empty line.